src/intervals/conversion_promotion.jl

```diff
@@ -2,14 +2,12 @@
 
 ## Conversion and promotion
 
-## Default conversion is to Interval{Float64}
+## Default conversion to Interval, corresponds to Interval{Float64}
 convert{T<:Real}(::Type{Interval}, x::T) = make_interval(Float64, x)
```

Comment:
I think the user has to be careful to specify to what kind of interval they want to convert. 
Alternatively, this information should be taken from the global `interval_parameters`